omod/src/main/webapp/pages/visit/templates/encounters/defaultHtmlFormEncounterLong.gsp

```diff
@@ -11,11 +11,9 @@
 <div class="content encounter-summary-long">
 
     <span ng-repeat="section in templateModel.sections" class="aligned">
-        <span ng-repeat="field in section.fields" class="aligned">
+        <span ng-repeat="field in (section.activeFields ? section.activeFields : section.fields)" class="aligned">
```

Comment:
For clarity, I'd be fine with just switching this to solely use the activeFields, with your new code I think if it ever got to "section.fields" is would be empty anyways?

Reply:
Thanks @mogoodrich ! I have removed the lines you indicated.